python/eos/analysis_file.py

```diff
@@ -220,3 +220,31 @@ def predictions(self):
         """Returns a list of all predictions recorded in the analysis file."""
 
         return self._predictions
+
+
+    def _repr_html_(self):
+        result = r'''
+        <table>
+            <colgroup>
+                <col width="100%" id="posterior"    style="min-width: 200px">
+            </colgroup>
+            <thead>
+                <tr>
+                    <th>posteriors</th>
+                </tr>
+            </thead>
+            <tbody>
+        '''
+        for p in self._posteriors:
+            result += fr'''
+                <tr>
+                    <td>{p}</td>
+                </tr>
+            '''
+
+        result += r'''
+            </tbody>
+        </table>
+        '''
```

Comment:
This only prints the name of the posteriors, right? What about the rest of the analysis file?

Reply:
For the time being, I'm happy with listing the available posteriors. I do not want to output each analysis, and right now we do not support creating "only" a log likelihood or log prior from an analysis file; only log posteriors.